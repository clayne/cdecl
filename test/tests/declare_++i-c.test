EXPECTED_EXIT=65
cdecl <<END
declare x of scope s as int
END
