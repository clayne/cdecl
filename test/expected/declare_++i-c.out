declare x of scope s as int
             ^
14: error: scoped names not supported in C
